x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/predicate/operator/arithmetic/Add.java

```diff
@@ -75,4 +82,9 @@ public static long processUnsignedLongs(long lhs, long rhs) {
     static double processDoubles(double lhs, double rhs) {
         return lhs + rhs;
     }
+
+    @Evaluator(extraName = "Datetimes", warnExceptions = { ArithmeticException.class, DateTimeException.class })
+    static long processDatetimes(long datetime, @Fixed TemporalAmount temporalAmount) {
+        return asMillis(asDateTime(datetime, DEFAULT_TZ).plus(temporalAmount));
```

Comment:
Why use UTC and not the request timezone?

Reply:
I've added a comment. 
Both timestamps that come from ES as well as those converted by our functions are in UTC.